tooling/validateSBOMcontent.sh

```diff
@@ -134,15 +134,25 @@ echo -n "Checking for JDK source SHA validity: "
 GITSHA=$(jq '.components[].properties[] | select(.name|test("OpenJDK Source Commit")) | .value' "$1" | tr -d \" | uniq)
 GITREPO=$(echo "$GITSHA" | cut -d/ -f1-5)
 GITSHA=$( echo "$GITSHA" | cut -d/ -f7)
-if ! git ls-remote "${GITREPO}" | grep "${GITSHA}"; then
-  echo "ERROR: git sha of source repo not found"
-  echo "GITREPO: ${GITREPO}"
-  echo "GITSHA: ${GITSHA}"
-  RC=1
+if [ -z "${EXPECTED_SCM_REF}" ]; then
+  commitData=$(curl --silent "https://api.github.com/repos/adoptium/jdk25u/commits/${GITSHA}")
```

Comment:
This is hard coded to jdk25u, and should probably just do a http header and http response check?
eg:
```
if ! curl --fail -I "$GIT_COMMIT_URL"; then
```